src/Umbraco.Core/UmbracoApplicationBase.cs

```diff
@@ -201,7 +203,10 @@ protected void Application_End(object sender, EventArgs evargs)
         {
             HandleApplicationEnd();
             OnApplicationEnd(sender, evargs);
-            LogManager.Shutdown();
+
+            //Not sure if we need to do this - as my POC approach I never had to deal with this
+            //As the LightInject container when tearing down will dispose of Serilog AFAIK
+            Log.CloseAndFlush();
```

Comment:
If it doesn't crash anything then might be better to just have it. Is it also IDisposable? i.e. would calling .Dispose() do the same thing?

Reply:
Yes the logger itself is IDisposable AFAIK and hence my comment/note to say that I am not 100% sure if we need to keep in this explicit call to `Log.CloseAndFlush()`